de/_posts/2019-11-30-js-testing-workshop.markdown

```diff
@@ -0,0 +1,53 @@
+---
+tags: intermediate js_testing_workshop
+categories: de events
+i18n_key: js_testing_workshop_2019_11
+---
+
+![](/assets/images/JStesting-Nov19.jpg)
+
+## Lerne noch mehr Programmieren ♥️
+
+Möchtest du JavaScript testing lernen? Dann ist JS testing Workshop der perfekte Tag für dich um weiter zu lernen.
+
+#### Wann: **30.11.2019, 09:30 - 16:00**
+#### Wo: **bitcrowd, Oranienstr 6, 10997 Berlin**
+
+#### Bewerben:
+Der Workshop richtet sich an Frauen* mit JavaScript Erfahrung. [**Hier kannst du dich bewerben!**](https://forms.gle/dwyLkQXRsnVHGxnx6)
+
+Bist du ein_e JavaScript Coach_in? [**Melde dich als Coach für den Workshop**](https://forms.gle/grVdR7y7hhZ9GCmu8)!
+
+#### Bewerbungs-Zeitplan
+- Beginn des Check-In: **13. November**
+- Ende des Check-In: **28. November**
+- Bestätigungs-E-Mails werden gesendet: **Es wird keine Bestätigungs-E-Mail geben. Wenn du dich anmeldest, erwarten wir dich beim Workshop.**
+
+
+## Workshop-Programm
+### Samstag, 30. November - Workshop Day
+- 09:30 - 10:00: **Kaffee & Anmeldung**
+- 10:00 – 10:10: **Willkommens-Intro** von code curious
+- 10:10 – 11:00: **Einführung** in JavaScript testing
+- 11:00 - 11:15: **Pause**
+- 11:15 - 12:00: **Vorbereitung für den praktischen Teil** in Gruppen
+- 12:00 – 13:00: **Mittagessen** (Es wird von bitcrowd bereitgestellt)
+- 13:00 - 13:20: **Lightning Talks**
+- 13:20 – 16:00: **Tests schreiben in Gruppen**
+
+## Was sollst du mitbringen?
+Du solltest einen Laptop mitbringen (vergiss Dein Ladekabel nicht). Du kannst gerne um 9 Uhr zum Workshop kommen, wenn du irgendwelche Probleme mit deinem Computer hast.
+
+## Veranstaltungsort
+
+Wir treffen uns im ShareNow-Büro. Adresse: Oranienstr 6, 10997 Berlin.
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2428.8534753300955!2d13.422867315900039!3d52.4998918798104!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47a84fac97094a8f%3A0x84c7ec5f6fb1f73!2sbitcrowd%20GmbH!5e0!3m2!1sen!2sde!4v1573605432282!5m2!1sen!2sde" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
+
+### Barrierefreiheit
+
+Der Veranstaltungsort ist rollstuhlgängig. Bitte schreib uns im Voraus eine [E-Mail](mailto:contact@codecurious.org), damit wir den Aufzugsschlüssel organisieren können.
```

Comment:
> rollstuhlgängig

Is that a commonly used word?

Reply:
I honestly don't know, the word is on Duden so I guess so 🤷‍♀